src/LayoutManager/LayoutManager.spec.ts

```diff
@@ -182,39 +179,24 @@ describe('Layout Manager', () => {
         expect(performLayout).not.toHaveBeenCalled();
       });
     });
-
-    it.each([
-      { trigger: LAYOUT_TYPE_INITIALIZATION, action: 'subscribe' },
-      { trigger: LAYOUT_TYPE_ADDED, action: 'subscribe' },
-      { trigger: LAYOUT_TYPE_REMOVED, action: 'unsubscribe' },
-    ] as {
-      trigger: LayoutTrigger;
-      action: 'subscribe' | 'unsubscribe';
-    }[])(
-      '$trigger trigger should $action targets and call target hooks',
-      ({ action }) => {
-        const lifecycle: jest.SpyInstance[] = [];
```

Comment:
this 3 tests were failing because i wrote bad code. And it was immediate to me to understand what they were testing for 2 reasons:
The connection between the array of data and the test variables force me to look up and down every time to understandand what is action and what is trigger, and the lifecycle array was a convoluted way to spell 3 times what has been calle, but without clearly saying with what has been called.

In general we should keep those array method for tests either for simple cases [true, false] or for large cases in which we need to use points or something else.

In the other cases i really prefer to spell clearly what the test is doing